mscore/exportxml.cpp

```diff
@@ -1156,12 +1156,12 @@ void ExportMusicXml::credits(Xml& xml)
       // determine page formatting
       const PageFormat* pf = _score->pageFormat();
       if (!pf) return;
-      const double h  = getTenthsFromInches(pf->size().height());
-      const double w  = getTenthsFromInches(pf->size().width());
-      const double lm = getTenthsFromInches(pf->oddLeftMargin());
-      const double rm = getTenthsFromInches(pf->oddRightMargin());
-      //const double tm = getTenthsFromInches(pf->oddTopMargin());
-      const double bm = getTenthsFromInches(pf->oddBottomMargin());
+      const double h  = pf->size().height()   * PageFormat::SCALE_XML;
```

Comment:
I don't get this. I think it's wrong. MusicXML needs all these values in 10th of space. Testing further.